backend/settings.py

```diff
@@ -277,4 +278,14 @@
     'SERVE_INCLUDE_SCHEMA': False,
 }
 
-DRF_TRACKING_ADMIN_LOG_READONLY = True
\ No newline at end of file
+DRF_TRACKING_ADMIN_LOG_READONLY = True
+
+# https://django-q2.readthedocs.io/en/master/configure.html
+Q_CLUSTER = {
+    'name': 'CCM_Cluster',
+    'workers': int(os.getenv('Q_CLUSTER_WORKERS', 4)),
+    'timeout': int(os.getenv('Q_CLUSTER_TIMEOUT', 1800)), # 30 minutes
+    'retry': int(os.getenv('Q_CLUSTER_RETRY', 3600)), # 1hr
```

Comment:
We could adjust the `timeout` and `retry` after when we get some number on how long the task for enrolling user takes, probably based on the Enrolling U-M user we be used as benchmark since we will executing  1000+ enrollment per request